web/.vuepress/components/HeroBlock.vue

```diff
@@ -78,7 +78,7 @@ import Waves from '../util/waves';
 import { version } from '../../../package.json';
 
 export default {
-    name: 'Hero',
+    name: 'HeroBlock',
```

Comment:
Renamed custom single-word components due to `vue/multi-word-component-names` eslint rule: https://eslint.vuejs.org/rules/multi-word-component-names.html